src/modules/showTask.js

```diff
@@ -0,0 +1,71 @@
+import taskArr from './toDo.js';
+import * as Store from './store.js';
+import removeTask from './remove.js';
+
+const taskList = document.getElementById('task-list');
+
+const changeDesc = (input, taskel, el) => {
```

Comment:
Please, I noticed the "el" parameter is not that descriptive, maybe you can change it to a better descriptive name.